artemis-commons/src/main/java/org/apache/activemq/artemis/utils/DefaultSensitiveStringCodec.java

```diff
@@ -139,8 +140,12 @@ private void updateKey(String key) {
       public String decode(String secret) throws Exception {
          SecretKeySpec key = new SecretKeySpec(internalKey, "Blowfish");
 
-         BigInteger n = new BigInteger((String) secret, 16);
-         byte[] encoding = n.toByteArray();
+         byte[] encoding;
+         try {
+            encoding = new BigInteger(secret, 16).toByteArray();
+         } catch (Exception ex) {
+            throw new IllegalArgumentException("Password must be encrypted.");
```

Comment:
I think it would also be worth logging the original exception at debug level.  Be sure to wrap the log statement in `if (logger.isDebugEnabled())`.